src/components/DropDownMenu.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  export let label: string;
+  export let id: string;
+  export let options: Record<string, string>;
+  export let selected: string;
+  export let placeholder: string;
+  export let onChange = () => {};
+  export let onSelect = () => {};
+  export let required = false;
+</script>
+
+<style>
+    option[value=""][disabled] {
+      display: none;
+    }
+</style>
+
+<label class="section-title" for={id}>{label}</label>
+<div style="margin-bottom: 14px;"></div>
```

Comment:
Apply margin bottom to the <label /> and delete this so we do not have any extra or empty elements. Can also be defined using tailwind `mb-3.5` is the equivalent. Since we're using tailwind, should avoid using the style tag.